Makefile

```diff
@@ -126,6 +128,13 @@ doc: $(GLOBFILES) $(VFILES)
 	$(COQDOC) -toc $(COQDOCFLAGS) -html $(COQDOCLIBS) -d $(ENHANCEDDOCTARGET) \
 	--with-header $(ENHANCEDDOCSOURCE)/header.html $(VFILES)
 	sed -i'.bk' -f $(ENHANCEDDOCSOURCE)/proofs-toggle.sed $(ENHANCEDDOCTARGET)/*html
+latex: Makefile $(GLOBFILES) $(VFILES)
+	mkdir -p $(LATEXTARGET)
+	$(COQDOC) -toc $(COQDOCFLAGS) -latex $(COQDOCLATEXOPTIONS) $(COQDOCLIBS) -utf8 -d $(LATEXTARGET) $(VFILES)
+pdf: latex
+	cd $(LATEXTARGET) ;\
+	latexmk -pdf *.tex
```

Comment:
It would be better to get the names of the tex files by processing $(VFILES), since there could be old
*.tex files lying about if file names have been changed.